src/__docs__/components/BreakpointMethods.js

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { BreakpointHoc } from '../../Breakpoint';
+
+const Text = styled.span`
+  transition: background 0.23s;
+  background: ${props => (props.active ? '#a0ffa9' : 'transparent')};
+`;
+
+const breakpoints = [
+  { name: 'small', maxWidth: 600 },
+  { name: 'medium', minWidth: 'small', maxWidth: 'large' },
+  { name: 'large', minWidth: 1200, exact: true },
+];
+
+const keys = ['isLt', 'isLte', 'isGt', 'isGte', 'isEq'];
+
+function Highlight({ children }) {
+  const [isFlashing, setFlashing] = React.useState(false);
+  const initialMount = React.useRef(true);
+
+  React.useEffect(() => {
+    if (initialMount.current) {
+      initialMount.current = false;
+    } else {
+      setFlashing(true);
+      setTimeout(() => setFlashing(false), 1000);
+    }
+  }, [children]);
+
+  return <Text active={isFlashing}>{children}</Text>;
+}
+
+function BreakpointMethods({ bp }) {
+  const results = breakpoints.map(({ name }) => (
+    <div key={name}>
+      {keys.map(key => (
+        <div key={key}>
+          <code>{`${key}('${name}')`}</code>
+          <Highlight>{String(bp[key](name))}</Highlight>
```

Comment:
Needs some kind of separator 
```jsx
<code>{`${key}('${name}') === `}</code>
```

<details>

<summary>Before/After</summary>


![before](https://user-images.githubusercontent.com/936069/66414451-4acc6000-e9ae-11e9-9271-8f3c32c11675.png)


![after](https://user-images.githubusercontent.com/936069/66414518-69325b80-e9ae-11e9-8df5-b729fca3a949.png)


</details>